flake.nix

```diff
@@ -27,14 +27,24 @@
               hash = "sha256-eKYQq9OelOD5E+nuXNoehbtizWM1U97LngDT2SAQGc4=";
             };
           });
+          cbmc = pkgs.cbmc.overrideAttrs (old: rec {
+            version = "731338d5d82ac86fc447015e0bd24cdf7a74c442";
+            src = pkgs.fetchFromGitHub {
+              owner = "diffblue";
+              repo = "cbmc";
+              rev = "731338d5d82ac86fc447015e0bd24cdf7a74c442";
```

Comment:
I think you can also reference version by ${version} here

Reply:
Changed. Thanks